core/src/main/resources/schema/oracle/oracle-number-boolean-migration.sql

```diff
@@ -0,0 +1,6 @@
+-- SPDX-License-Identifier: Apache-2.0
+
+-- see https://github.com/apache/pekko-persistence-jdbc/pull/323
+
+ALTER TABLE EVENT_JOURNAL MODIFY "deleted" NUMBER(1)
```

Comment:
Shouldn't this be `DELETED`? Also, by altering the column in the table, you lose all the checks [already present in the column](https://github.com/apache/pekko-persistence-jdbc/pull/323/files#diff-b43ed855a228bcb11465dc5b9ad2769e9a84db0f71d280928a1dd133a78efaf3R6)